src/wizard/Step1/components/RadioCard.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
Let combine both `src\wizard\Step1\components\RadioCard.vue` and `src\wizard\Step1\components\RadioCards.vue1`  into a single component, rename it to `ATATRadioButtonCard.vue` and put it in this folder `C:\Users\TonySumner\Documents\atat-web-ui\src\components`.   This will make it available globally if we need it later.

You will have to add it a reference to it in`src\main.ts` file.